recipes/r-convertr/meta.yaml

```
@@ -0,0 +1,74 @@
+{% set version = '0.2.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-convertr
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://github.com/SantanderMetGroup/convertR/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: ad56181158a7d6f7967ffb323ccd8f3dcc8cf3343f354787d9be3b20a28bd426
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}git
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base >=3.6,<4
+    - r-magrittr
+    - r-transformer >=1.7.2
+    - r-udunits2
+  run:
+    - r-base >=3.6,<4
+    - r-magrittr
+    - r-transformer >=1.7.2
+    - r-udunits2
+
+test:
+  commands:
+    - $R -e "library('convertR')"           # [not win]
+    - "\"%R%\" -e \"library('convertR')\""  # [win]
+
+about:
+  home: https://github.com/SantanderMetGroup/climate4R
+  summary: Implements different variable derivations and unit transformations exploiting the
+    UDUNITS-2 capabilitites for the climate4R CDM (<http://meteo.unican.es/climate4R>).
+  doc_url: https://github.com/SantanderMetGroup/convertR
+  dev_url: https://github.com/SantanderMetGroup/convertR
+  license: GPL-3.0-or-later
+  license_family: GPL3
+  license_file: LICENSE.txt
```

If the license is not part of the source tarball, no need to include it here as the r-base package provides it and it can be used here (that is they way how CRAN packages are doing it by relying on the R installation):
```suggestion
  license_file:
    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
```

---

this also applies to the other open PRs

---

I have had include LICENSE.txt because the conda skeleton from CRAN didn't include your line addition.